security-acceptance-tests/security-reports/security-report.html

```diff
@@ -0,0 +1,1820 @@
+<html>
```

Comment:
I'm guessing that this report should not have been checked in. Any reports / outputs should not be included in source.